contracts/test/utils/Tokens.sol

```diff
@@ -0,0 +1,49 @@
+// SPDX-License-Identifier: MIT
+pragma solidity 0.7.6;
+
+import "forge-std/Test.sol";
+import "@openzeppelin/contracts/token/ERC20/IERC20.sol";
+import "./Addresses.sol";
+
+contract Tokens is Test {
+    IERC20 public weth;
+    IERC20 public usdt;
+    IERC20 public usdc;
+    IERC20 public dai;
+    IERC20 public wbtc;
+    IERC20 public lon;
+    IERC20 public ankreth;
+    IERC20[] public tokens;
+
+    constructor() {
+        if (vm.envBool("DEPLOYED")) {
+            // load ERC20s using address in env vars
+            weth = IERC20(vm.envAddress("WETH_ADDRESS"));
+            usdt = IERC20(vm.envAddress("USDT_ADDRESS"));
+            usdc = IERC20(vm.envAddress("USDC_ADDRESS"));
+            dai = IERC20(vm.envAddress("DAI_ADDRESS"));
+            wbtc = IERC20(vm.envAddress("WBTC_ADDRESS"));
+            lon = IERC20(vm.envAddress("LON_ADDRESS"));
+            ankreth = IERC20(vm.envAddress("ANKRETH_ADDRESS"));
+        } else {
+            // load ERC20s using constant address
+            weth = IERC20(WETH_ADDRESS);
+            usdt = IERC20(USDT_ADDRESS);
+            usdc = IERC20(USDC_ADDRESS);
+            dai = IERC20(DAI_ADDRESS);
+            wbtc = IERC20(WBTC_ADDRESS);
+            lon = IERC20(LON_ADDRESS);
+            ankreth = IERC20(ANKRETH_ADDRESS);
```

Comment:
These should be freshly deployed otherwise they are the same as `DEPLOYED` and RFQ don't have to deploy them in `setUp()`.

Reply:
For `DEPLOYED` case, it may be any network from testnet to L2 and therefore is different from forked mainnet. We can generalize the forked testing and always provide addresses before running a forked test but may cost another setup step when doing forked mainnet test.

Reply:
Yeah I forgot these are also used for forked tests...

I will write a setup that differentiate between different networks and setup tokens/system contracts accordingly.